sencha-workspace/packages/slate-cbl/src/view/skill/OverviewBody.js

```diff
@@ -0,0 +1,92 @@
+/*jslint browser: true, undef: true *//*global Ext*/
+Ext.define('Slate.cbl.view.skill.OverviewBody', {
+    extend: 'Ext.container.Container',
+    xtype: 'slate-cbl-skill-overviewbody',
+    requires: [
+        'Slate.cbl.model.Skill',
+
+        'Ext.form.field.ComboBox',
+        'Ext.data.ChainedStore'
+    ],
+
+    items: [
+        {
+            reference: 'skillStatement',
+
+            xtype: 'component',
+            autoEl: 'p',
+            padding: '16 32 0'
+        },
+        {
+            reference: 'demonstrationsTable',
+
+            xtype: 'component',
+            minHeight: 200,
+            tpl: [
+                '<table class="skill-grid">',
+                    '<thead>',
+                        '<tr class="skill-grid-header-row">',
+                            '<th class="skill-grid-header skill-grid-demo-index">&nbsp;</th>',
+                            '<th class="skill-grid-header skill-grid-demo-date">Date</th>',
+                            '<th class="skill-grid-header skill-grid-demo-level">Level</th>',
+                            '<th class="skill-grid-header skill-grid-demo-experience">Experience</th>',
+                            '<th class="skill-grid-header skill-grid-demo-context">Context</th>',
+                            '<th class="skill-grid-header skill-grid-demo-task">Performance&nbsp;Task</th>',
+                            // '<th class="skill-grid-header skill-grid-demo-artifact">Artifact</th>',
+                        '</tr>',
+                    '</thead>',
+                    '<tpl if="values && values.length">',
+                        '<tpl for=".">',
+                            '<tr class="skill-grid-demo-row" data-demonstration="{ID}">',
+                                '<td class="skill-grid-demo-data skill-grid-demo-index">{[xindex]}</td>',
+                                '<td class="skill-grid-demo-data skill-grid-demo-date">{Demonstration.Demonstrated:date("M j, Y")}</td>',
+                                '<td class="skill-grid-demo-data skill-grid-demo-level"><div class="level-color cbl-level-{Level}"><tpl if="Level==0">M<tpl else>{Level}</tpl></div></td>',
```

Comment:
This is getting pretty hard to read. I'd like to see the child elements and template logic formatted differently.